docsource/development.rst

```diff
@@ -16,7 +16,7 @@ steps are not necessary):
 - end-migration.py
 
 Since version 14, these files can be found here:
-``openupgrade-framework/openupgrade_scripts/scripts/<module-name>``
+``openupgrade_scripts/scripts/<module-name>``
```

Comment:
Now that you are here, it seems the initial intention was to put this into a new paragraph, but for proper RST syntax for that, you need to leave an extra empty line (the same for both changes).